### PR TITLE
Add rbf text option to transaction row

### PR DESCRIPTION
**Pull request by peachbits**
### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [x] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203444039875736

### CODE REVIEW

**Comment by peachbits** [comment]
Simulator Screen Shot - iPod touch (7th generation) - 2023-01-17 at 00 21 54 #2

File: src/components/themed/TransactionRow.tsx [hunk 1]
```diff
@@ -45,6 +47,11 @@ const TransactionRowComponent = (props: Props) => {
   const theme = useTheme()
   const styles = getStyles(theme)
 
+  const account = useSelector(state => state.core.account)
+  const currencyWallets = useWatch(account, 'currencyWallets')
+  const wallet = currencyWallets[transaction.walletId]
+  const canReplaceByFee = !!wallet.currencyInfo.canReplaceByFee
```
Comment (swansontec) [inline src/components/themed/TransactionRow.tsx:53]: Maybe `const { canReplaceByFee = false } = wallet.currencyInfo`?

File: src/locales/en_US.ts [hunk 2]
```diff
@@ -192,6 +192,7 @@ const strings = {
   fragment_transaction_list_tx_dropped: 'Dropped',
   fragment_transaction_list_tx_synchronizing: 'Synchronizing',
   fragment_transaction_list_confirmation_progress: '%s of %s Confirmations',
+  fragment_transaction_list_unconfirmed_rbf: 'Awaiting confirmation, may be replaced',
```
Comment (swansontec) [inline src/locales/en_US.ts:195]: I think Michael wants wording like "may be cancelled". We want normal people to understand that "you may not get the money". The word "cancelled" does this better than "replace".

**Review by swansontec — CHANGES_REQUESTED** [review]

File: src/locales/en_US.ts (hunk 2 above)
Comment (peachbits) [reply src/locales/en_US.ts]: Description says cancelled, his comment says replaced. I agree cancelled is better so I changed it

**Review by swansontec — APPROVED** [review]

**Pull request merged**